fe/fe-core/src/main/java/org/apache/doris/nereids/rules/RuleSet.java

```diff
@@ -39,6 +40,10 @@ public class RuleSet {
             .add(new JoinLeftAssociative())
             .build();
 
+    public static final List<Rule<Plan>> REWRITE_RULES = planRuleFactories()
```

Comment:
Is it appropriate to put aggregation and splitting at the current stage? 
In theory almost all rewrite rules don't care if agg is one or two. 
If the agg split is performed too early, it will affect the writing of some agg related rules.